css/index.css

```diff
@@ -143,4 +196,42 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+hr {
+	 margin-bottom: 20px;
+	margin-top:30px;
+    border-bottom: 1px dashed black;
+}
+
+
+
+.section2-3 {
+	display:flex;
+	
+	justify-content: flex-start;
+}
+
+.hr2 {
+	width:90%;
+	margin-right:10%
+}
+
+button {
+	margin-top:20px;
+	background-color: white; 
+  border: 1px solid black;
+  color: black;
+  padding: 20px;
+	padding-top: 10px;
+	padding-bottom:10px;
```

Comment:
The above three lines work, but a shorter way to say the same thing is `padding: <top/bottom> <left/right>;`. The two value shorthand for padding represent "top/bottom" and "left/right" respectively. Remember you also have the four value shorthand `padding: <top> <right> <bottom> <left>;`

In this case, that'd be `padding: 10px 20px;`